ai-assistants/loaders/messages.ts

```diff
@@ -14,32 +14,69 @@ export interface Props {
 export const Tokens = {
   POSITIVE: "@",
   NEGATIVE: "#",
+  OPTIONS: "&",
 };
+
 const normalize = (strContent: string) => {
-  return strContent.endsWith(Tokens.POSITIVE) ||
-      strContent.endsWith(Tokens.NEGATIVE)
-    ? strContent.slice(0, strContent.length - 2)
-    : strContent;
+  const hasOptions = strContent.includes(Tokens.OPTIONS);
```

Comment:
A category name, product name, can have ampersand inside it, so that code may break.

Maybe change it to `&&&`. Or a more complex token.